aws/resource_aws_ecs_service_test.go

```diff
@@ -494,6 +494,15 @@ func TestAccAWSEcsService_withPlacementStrategy(t *testing.T) {
 					resource.TestCheckResourceAttr("aws_ecs_service.mongo", "ordered_placement_strategy.0.field", "memory"),
 				),
 			},
+			{
+				Config: testAccAWSEcsServiceWithRandomPlacementStrategy(clusterName, tdName, svcName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckAWSEcsServiceExists("aws_ecs_service.mongo", &service),
+					resource.TestCheckResourceAttr("aws_ecs_service.mongo", "ordered_placement_strategy.#", "1"),
+					resource.TestCheckResourceAttr("aws_ecs_service.mongo", "ordered_placement_strategy.0.type", "random"),
+					resource.TestCheckNoResourceAttr("aws_ecs_service.mongo", "ordered_placement_strategy.0.field"),
```

Comment:
This seems to be failing acceptance testing at the moment:

```go
--- FAIL: TestAccAWSEcsService_withPlacementStrategy (32.95s)
    testing.go:538: Step 2 error: Check failed: Check 4/4 error: aws_ecs_service.mongo: Attribute 'ordered_placement_strategy.0.field' found when not expected
```

Reply:
Whoops, my mistake. Fixed and learned how to run the acceptance tests. :)

We want Field attribute to be nil when creating the resource, but the attribute still exists when querying the resource.